packages/core-mobile/app/services/notifications/NotificationsService.ts

```diff
@@ -321,16 +321,14 @@ class NotificationsService {
         pressAction: {
           id: PressActionId.OPEN_PORTFOLIO,
           launchActivity: LAUNCH_ACTIVITY
-        }
+        },
+        channelId: channelId ?? ''
```

Comment:
Is there a difference between passing an empty string and undefined for channelId?

Reply:
yes, in my experience `notifee` will report such payload as invalid

Reply:
it's weird since `channelId` is an optional parameter.

Reply:
it's optional for ios platform, on android it's required